ompy/models.py

```diff
@@ -391,6 +391,12 @@ class NormalizationParameters(Model):
     """Storage for normalization parameters + some convenience functions
     """
 
+    #: Element number of the nucleus
+    Z: Optional[int] = field(default=None,
+            metadata="Element number of the nucleus")  # noqa
+    #: Mass number of the nucleus
+    A: Optional[int] = field(default=None,
+            metadata="Mass number of the nucleus")  # noqa
```

Comment:
Currently, A and Z would be needed in the `spincutPars` for several of the spincut formulas. Would you suggest that we implement a check whether A and Z are available in spincutPars, and potentially otherwise get them from the class itself and vice versa?

Reply:
> Currently, A and Z would be needed in the `spincutPars` for several of the spincut formulas. Would you suggest that we implement a check whether A and Z are available in spincutPars, and potentially otherwise get them from the class itself and vice versa?

Thats a good idea, however I think that there shouldn't be a strict consistency between the A and Z in `spincutPars` and the attributes I added since there could be cases where one would like to use a different A and Z for the `spincutPars` than the actual A and Z of the nucleus. To me it make more sense issue a warning to the user that they are inconsistent, rather than failing or even worse; silently 'fixing'.

Reply:
Sounds good. Could just put it in the log info stream

Reply:
(or with the log level WARNING)

Reply:
I've updated now to give warnings to the user if there are inconsistencies between the `mass` set in `spincutPars` and `A` set in `NormalizationParameters`. These warnings are triggered when setting these attributes. The `A` attribute will default to the mass set in `spincutPars` if A is not explicitly set.